tools/embark-oss/README.md

```diff
@@ -8,10 +8,16 @@ This tool is used to validate that our open source projects adhere to our open s
 cargo run check-maintainers
 ```
 
-This command checks to see if every project listed in the Embark [opensource-website data.json][data.json] has an official maintainer listed in its `CODEOWNERS` file within its repository.
+This command checks to see if every project listed in the Embark [opensource-website data.json][data.json] conforms to our open source guidelines to the extent that this tool can detect.
 
 [data.json]: https://github.com/EmbarkStudios/opensource-website/blob/main/data.json
 
+```shell
+cargo run check-maintainers PROJECT_REPO_NAME
```

Comment:
@lpil what's the high-level need for this? Can you please explain? :)

Reply:
@lpil shouldn't this be validate-all?

Reply:
Well spotted!

Reply:
@soniasingla I used this command to quickly check the status of new projects that had not been added to the data.json yet. It could be used any time you might want to inspect a single project.